### DISCUSSION TITLE
Auth Proxy: Support forwarding of JWT token to plugins passed to Grafana through the auth proxy

### DISCUSSION THREAD

**Discussion by kostrse** [grafana/grafana#62614] [Plugins]
In case if user was authenticated using auth proxy and ID token header was passed from the proxy, then pass the received ID token to Grafana plugins via PluginContext.User struct.

This is Auth Proxy implementation task for:
* https://github.com/grafana/grafana-plugin-sdk-go/issues/506

**Comment by marefr** [comment]
Converted this to a discussion since this is a new feature request and not currently supported.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
